src/types.deno.ts

```diff
@@ -139,12 +139,14 @@ export class InputFile {
     }
 }
 
-async function* fetchFile(url: string | URL): AsyncIterable<Uint8Array> {
+async function fetchFile(
+    url: string | URL,
+): Promise<AsyncIterable<Uint8Array>> {
     const { body } = await fetch(url);
     if (body === null) {
         throw new Error(`Download failed, no response body from '${url}'`);
     }
-    yield* body;
+    return body;
```

Comment:
Rewrite `fetchFile` methods, because `ReadableStream` extends `AsyncIterator` function allowing you to use `yield * ReadableStream` in other methods.

Reply:
very cool